repos/system_upgrade/el7toel8/actors/networkmanagerupdateclientid/actor.py

```diff
@@ -27,7 +28,7 @@ def process(self):
 
             try:
                 r = run(['nm-update-client-ids.py'], split=True)['stdout']
-            except (OSError, CalledProcessError) as e:
```

Comment:
The `OSError` exception can occur as well currently. So this doesn't seem like correct change now.

Reply:
@vojtechsokol, will you be addressing this?